configs/projects/puppet-agent.rb

```diff
@@ -22,11 +22,11 @@
       proj.setting(:RememberedInstallDirRegKey, "RememberedInstallDir")
     end
     proj.setting(:links, {
-        :HelpLink => "http://links.puppetlabs.com/customer-support-foss",
-        :CommunityLink => "http://links.puppetlabs.com/windows-installer-starting-out",
-        :ForgeLink => "http://links.puppetlabs.com/forge-windows",
-        :NextStepLink => "http://links.puppetlabs.com/windows-installer-next-steps-foss",
-        :ManualLink => "http://links.puppetlabs.com/windows-manual-foss",
+        :HelpLink => "http://puppet.com/services/customer-support",
+        :CommunityLink => "http://docs.puppet.com",
+        :ForgeLink => "http://forge.puppet.com",
+        :NextStepLink => "http://docs.puppet.com",
```

Comment:
So previously `windows-installer-next-steps-pe` would be going to https://docs.puppet.com/pe/latest/quick_start_install_agents_windows.html 

Since there is no longer a distinction between PE and FOSS, I think that it should go there as well.

Other reasonable options:

* https://docs.puppet.com/puppet/latest/reference/install_windows.html
* https://docs.puppet.com/puppet/latest/reference/services_agent_windows.html
* https://docs.puppet.com/puppet/latest/reference/services_commands_windows.html